samples/word/retrieve_comments/cs/Program.cs

```diff
@@ -1,18 +1,21 @@
-#nullable disable
-
 using DocumentFormat.OpenXml.Packaging;
 using DocumentFormat.OpenXml.Wordprocessing;
 using System;
 
+GetCommentsFromDocument(args[0]);
+
 static void GetCommentsFromDocument(string fileName)
 {
-    using (WordprocessingDocument wordDoc =
-        WordprocessingDocument.Open(fileName, false))
+    using (WordprocessingDocument wordDoc = WordprocessingDocument.Open(fileName, false))
     {
-        WordprocessingCommentsPart commentsPart =
-            wordDoc.MainDocumentPart.WordprocessingCommentsPart;
+        if (wordDoc.MainDocumentPart is null || wordDoc.MainDocumentPart.WordprocessingCommentsPart is null)
+        {
+            throw new System.NullReferenceException("MainDocumentPart and/or WordprocessingCommentsPart is null.");
```

Comment:
instead of nullreferenceexception, should throw InvalidOperation

Reply:
Done